spec/hyacinth/adapters/digital_object_search_adapter/solr_spec.rb

```diff
@@ -60,12 +60,10 @@
 
   context "#solr_document_for" do
     let(:authorized_object) { FactoryBot.build(:digital_object_test_subclass, :with_sample_data) }
-    let(:publication_adapter) { Hyacinth::Adapters::PublicationAdapter::Abstract.new }
     let!(:external_identifier_adapter) { Hyacinth::Adapters::ExternalIdentifierAdapter::Memory.new }
 
     before do
       authorized_object.send :uid=, 'dummy-uid'
-      allow(Hyacinth::Config).to receive(:publication_adapter).and_return(publication_adapter)
       allow(Hyacinth::Config).to receive(:external_identifier_adapter).and_return(external_identifier_adapter)
```

Comment:
Hi, just wanted the background about why the publish_adapter is being replaced by the external_identifier_adapter here. Thanks.

Reply:
Not a replacement in this case, just a deletion of the upper of the two lines.  With the publication adapter config change I made for the test environment, we no longer need to fake the `Hyacinth::Config.publication_adapter` method call here, so that's why I took the line out.